### PR TITLE
(feat) copy circuit annotations

### PR DESCRIPTION
**Pull request by adria0**
This PR adds annotations to copy circuit

### CODE REVIEW

File: zkevm-circuits/src/copy_circuit.rs [hunk 1]
```diff
@@ -131,6 +131,18 @@ impl<F: Field> SubCircuitConfig<F> for CopyCircuitConfig<F> {
...
+        tx_table.annotate_columns(meta);
+        rw_table.annotate_columns(meta);
+        bytecode_table.annotate_columns(meta);
+        copy_table.annotate_columns(meta);
+
+        // annotate circuit specific columns
+        meta.annotate_lookup_any_column(is_last, || "is_last");
+        meta.annotate_lookup_any_column(value, || "value");
+        meta.annotate_lookup_any_column(is_code, || "is_code");
+        meta.annotate_lookup_any_column(is_pad, || "is_pad");
```
Comment (leolara) [inline zkevm-circuits/src/copy_circuit.rs:144]: @adria0 are all these, like is_pad lookup columns? If not perhaps they have to be annotated with `region.name_column`?

Comment (CPerezz) [reply zkevm-circuits/src/copy_circuit.rs:144]: To annotate columns that are used as lookup and don't have a `TableColumn` type. NOTE that within regions, these will be overwritten by the region annotations unless the error is a lookup one.

Comment (leolara) [reply zkevm-circuits/src/copy_circuit.rs:144]: so I see `is_pad` that is used in many gates and as part of the condition of lookups, but not like something that has to match in the lookup. Should we annotate `is_pad` the other way? In the synthesise with the region?

**Review by leolara — APPROVED** [review]

File: zkevm-circuits/src/copy_circuit.rs (hunk 1 above)
Comment (CPerezz) [reply zkevm-circuits/src/copy_circuit.rs]: > > To summarize, I agree with @leolara . The columns that are specific to the circuit and aren't used as lookup, should be annotated inside the `Region`.
> 
> IMO we should add this comment to the method documentation

Feel free to file an issue in our fork!! :)

**Review by CPerezz — APPROVED** [review]
LGTM

**Pull request merged**